hil-test/build.rs

```diff
@@ -34,6 +34,8 @@ fn main() -> Result<(), Box<dyn Error>> {
     let chip = Chip::from_str(device_name)?;
     let config = Config::for_chip(&chip);
 
+    esp_metadata::define_all_possible_symbols();
```

Comment:
I guess this could happen inside `define_symbols` so we will never forget to call it?

Reply:
Good idea! Thanks!